tasks/create.yaml

```diff
@@ -5,7 +5,7 @@ tasks:
       - task: pepr-build
 
       - description: "Create the UDS Core Standard Zarf Package"
-        cmd: "zarf package create packages/standard --confirm"
+        cmd: "zarf package create packages/standard --confirm  --flavor ${UDS_FLAVOR}"
```

Comment:
Wondering if we should put a default flavor to simplify the dev experience (i.e. set default `UDS_FLAVOR` to `upstream` so that local dev can remain the same as previous experience.

Reply:
I would like a default. I'll test locally to see if setting UDS_FLAVOR as a top level variable works as expected to facilitate this and update accordingly if it does, otherwise we can get that sorted on the uds cli side and then add it.